packages/gradle/src/plugin/nodes.ts

```diff
@@ -139,13 +134,17 @@ export const createNodes: CreateNodes<GradlePluginOptions> = [
       calculatedTargets[hash] = {
```

Comment:
Can this line be combined with the above line so we don't have to keep them in sync?

```
calculatedTargets[hash] = project
```

Reply:
changed